sdks/python/apache_beam/transforms/core.py

```diff
@@ -1222,6 +1225,67 @@ def __init__(self, fn, *args, **kwargs):
     from apache_beam.runners.common import DoFnSignature
     self._signature = DoFnSignature(self.fn)
 
+  def with_dead_letters(
```

Comment:
suggest: with_exception_handling deadletter is common term, but may not translate well non english speakers.

Reply:
Good point. Done.